teacher.rb

```diff
@@ -1,10 +1,8 @@
 require_relative './person'
 
-class Teacher
+class Teacher < person
```

Comment:
- The person is a class and should be spelled as  `Person`. please fix this before you merge

Reply:
thanks i did it